client/redux/user/index.js

```diff
@@ -53,11 +52,10 @@ export const login = (user) => async (dispatch) => {
   dispatch(logUserRequest(user))
 
   try {
-    await api.loginUser(user)
-
-    const token = getCookies('jwt')
+    const { data } = await api.loginUser(user)
+    const { token } = data
 
-    dispatch(getData(token))
+    dispatch(getData({ token }))
```

Comment:
@AStashevskaya Should we also change params for `client/redux/user/index.js` in `signIn` action?